.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -25,61 +25,79 @@ jobs:
   testing:
     needs: qa_docs
     strategy:
       # If a matrix fail, do NOT stop other matrix, let them run to completion
       fail-fast: false
       matrix:
-        os: [ "ubuntu-18.04", "ubuntu-20.04", "macos-10.15" ]
-        platform: [ "posix" ]
+        os: [ "ubuntu-18.04", "ubuntu-20.04", "macos-10.15", "windows-latest" ]
         python-version: [ "3.6", "3.7", "3.8", "3.9", "pypy3" ]
-        include:
-          - os: "windows-latest"
-            platform: "mswin"
-            # Only the latest frozen branch as indicated on
-            # https://devguide.python.org/#branchstatus
-            # ('frozen' means Status == "security")
-            # Reason being that Windows users can change their Python anytime,
-            # so there is no benefit in testing all released Python versions.
-            # Plus pypy3 implementation in Windows is ... complicated. We
-            # should stay away from all those complications; users choosing to
-            # run aiosmtpd on pypy3 on Windows should be considered advanced
-            # and know what they're doing.
-            python-version: "3.7"
     runs-on: ${{ matrix.os }}
     steps:
     - name: "Checkout latest commit"
       uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # Required by codecov/codecov-action@v1
     - name: "Set up Python ${{ matrix.python-version }}"
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
     - name: "Install dependencies"
+      # language=bash
       run: |
-        python -m pip install --upgrade pip
-        pip install tox
-    - name: "Execute tox"
-      env:
-        PLATFORM: ${{ matrix.platform }}
-      # "py" testenv means we'll use whatever Python provided by GA,
-      # as specified in 'matrix' above. No need to specify exact version.
+        python -m pip install --upgrade pip setuptools wheel
+        # Test deps
+        pip install colorama coverage[toml] packaging pytest pytest-cov pytest-mock
+        # Package deps
+        python setup.py develop
+    - name: "Execute testing"
+      shell: bash
+      # language=bash
       run: |
-        tox -e py-cov
+        if [[ ${{ matrix.python-version }} == 3.6 ]]; then
+          export PY_36="eq"
+        else
+          export PY_36="gt"
+        fi
+        if [[ ${{ matrix.os }} == windows-latest ]]; then
+          export PLATFORM="mswin"
+        else
+          export PLATFORM="posix"
+        fi
+        pytest --cov --cov-report=xml --cov-report=term
     - name: "Report to codecov"
       # Ubuntu 18.04 came out of the box with 3.6, and LOTS of system are still running
       # 18.04 happily, so we choose this as the 'canonical' code coverage testing.
       # One day we'll have to revisit this and bump the version ...
-      if: always() && matrix.python-version == '3.6' && matrix.os == 'ubuntu-latest'
+      if: matrix.python-version == '3.6' && matrix.os == 'ubuntu-18.04'
```

Comment:
Explicit pointing to ubuntu-18.04 because ubuntu-latest is transitioning to 20.04